prql-js/Cargo.toml

```diff
@@ -1,12 +1,18 @@
 [package]
 description = "Javascript bindings for prql-compiler"
-edition = "2021"
-license = "Apache-2.0"
 name = "prql-js"
 publish = false
-repository = "https://github.com/PRQL/prql"
+
+edition.workspace = true
+rust-version.workspace = true
 version.workspace = true
```

Comment:
btw
```toml
[package.metadata.release]
shared-version = true
```
is implicit when you use `version.workspace = true`

Reply:
Great, thanks, changed!